File: lib/data/non/ruby/input/accelerometer.rb
class Accelerometer
  java_import 'com.badlogic.gdx.Gdx'

  def x
    Gdx.input.getAccelerometerX()
  end
  
  def y
    Gdx.input.getAccelerometerY()
  end
  
  def z
    Gdx.input.getAccelerometerZ()
  end
  
  def rotation
    x, y, z
  end
  
  def orientation
    Gdx.input.getRotation()
  end
end

File: lib/data/non/lua/input/accelerometer.lua
non.accelerometer = {}

local Gdx = NON_GDX

function non.accelerometer.x()
  return Gdx.input:getAccelerometerX()
end
  
function non.accelerometer.y()
  return Gdx.input:getAccelerometerY()
end
  
function non.accelerometer.z()
  return Gdx.input:getAccelerometerZ()
end
  
function non.accelerometer.rotation()
  return Gdx.input:getRotation()
end